lib/ParseSIL/ParseSIL.cpp

```diff
@@ -885,6 +887,119 @@ void SILParser::convertRequirements(SILFunction *F,
   }
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+/// Parse a SIL constant value (one of the following categories):
+/// - An integer datatype and literal (i32 42).
+/// - A floating point datatype and literal (f64 3.14).
+/// - A metatype (the instance type is parsed) ($Float).
+/// - An aggregate ([1, 2, 3]).
```

Comment:
Comment should be updated to include types, e.g.:  [i32 1, i64 2, f32 3]

Reply:
I believe the comment already included types.
I added some more for extra clarity:
```diff
 // SWIFT_ENABLE_TENSORFLOW
 /// Parse a SIL constant value (one of the following categories):
-/// - An integer datatype and literal (i32 42).
-/// - A floating point datatype and literal (f64 3.14).
+/// - An integer datatype and literal (i32 -3, i64 42).
+/// - A floating point datatype and literal (f32 3, f64 3.14).
 /// - A metatype (the instance type is parsed) ($Float).
 /// - An aggregate ([1, 2, 3]).
 /// Returns true on error.
```

Reply:
I saw you address my comment above by editing your comment. Marking to resolve.